generators/app/templates/src/app/components/CustomText/styles.js

```diff
@@ -0,0 +1,55 @@
+import { StyleSheet } from 'react-native';
+
+import { blue, white, green, gray, black, transparent } from '../../../utils/colors';
+import { IS_SMALL_DEVICE } from '../../../utils/constants';
```

Comment:
Mmm, `utils/constants` can become a `god file`. I'll raise an issue to address this refactor.